### ISSUE TITLE
Dial code not updated using setNumber

### ISSUE THREAD

**Issue by Dr3am3rz** [jackocnr/intl-tel-input#1138]
I have a registration form, there is no issue getting the selected country dial code + phone number e.g. +651234567 and insert into database.
When I tried to retrieve +651234567 and do a iti.setNumber("+651234567"), it shows perfectly fine in the input field. But when I tried to update the number, the country dial code will always be the first country in the list no matter which country I select.

Not sure if anyone here can help me on this or there's really wrong somewhere in the plugin.

### Steps to reproduce
1.  
2.  
3.  

### Expected behaviour
Country flag and dial code should be updated or selected accordingly

### Actual behaviour
It always set the first country no matter which country you have selected

### Initialisation options
List any options you're using e.g. utilsScript or preferredCountries

hiddenInput: "full_number"
utilsScript: "/js/utils.js"

**Comment by jackocnr** [comment]
If you can find the minimal set of code that reproduces the problem and put that in a CodePen, then I'd be happy to take a look. Here is [an example pen](https://codepen.io/jackocnr/pen/yehvE) with the plugin up and running - feel free to fork this if it helps.

**Comment by Dr3am3rz** [comment]
Hi Jack,

Appreciate for your help!

I realized what went wrong when I tried to replicate my problem in the
codepen. I runned the plugin again somewhere in my code, therefore the
plugin was run twice and there are 2 country list dropdown. End up I am
changing visually on the 2nd runned plugin but the data was sent using the
1st runned plugin. Stupid me *facepalm*

Sorry to have bothered you. Have a nice weekend! =)


Regards,
Jeffrey
